_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -86,7 +84,8 @@ This `autocomplete` [attribute value][] only has the required token, and is vali
 
 ```html
 <form autocomplete="off">
-	<select autocomplete="bday-month">
+	<label for="bday">Birthday month</label>
```

Comment:
I think we should stick to implicit labels (`<label>Foo <widget>…</widget> </label>`) rather than explicit ones (`<label for="…">`) because this is not the core of the rule (this is not a rule about labels, so no need to show all ways to give a label), all the existing cases are using implicit labels, and it limits errors in IDs.